docs/src/design/stack/io_ops.md

```diff
@@ -68,16 +68,15 @@ $$
 Using the above variable, we define the value representing the memory access request as follows:
 
 $$
-u_{mem} = \alpha_0 + \alpha_1 \cdot op_{mem\_read} + \alpha_3 \cdot s_0 + \alpha_4 \cdot clk + v
+u_{mem} = \alpha_0 + \alpha_1 \cdot op_{mem\_read} + \alpha_2 \cdot ctx + \alpha_3 \cdot s_0 + \alpha_4 \cdot clk + v
 $$
 
 In the above:
 - $op_{mem\_read}$ is the unique [operation label](../chiplets/main.md#operation-labels) of the memory read operation.
+- $ctx$ is the current context value for the memory.
```

Comment:
Could we change this to: "$ctx$ is the identifier of the current memory context."